pkg/controller/v1beta1/inferenceservice/reconcilers/kubernetes/kube_reconciler.go

```diff
@@ -0,0 +1,387 @@
+/*
+Copyright 2020 kubeflow.org.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package native
+
+import (
+	"context"
+
+	"github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1"
+	v1beta1api "github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1"
+	"github.com/kubeflow/kfserving/pkg/constants"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	exv1beta1 "k8s.io/api/extensions/v1beta1"
```

Comment:
this package is going to be deprecated check https://github.com/kubernetes/kubernetes/pull/74057/files

Reply:
ok, get it